cargo-concordium/src/build.rs

```diff
@@ -451,6 +451,37 @@ fn write_schema_json(
     Ok(())
 }
 
+/// Write the template of the schema to a file or print it
+/// to the console if `out` is None.
+pub fn write_schema_template(
+    out: Option<PathBuf>,
+    schema: &VersionedModuleSchema,
+) -> anyhow::Result<()> {
+    match out {
+        // writing the template of the schema to a file
+        Some(out) => {
+            println!(
+                "   Writing the template of the schema to {}.",
+                out.display()
+            );
+            if let Some(out_dir) = out.parent() {
+                fs::create_dir_all(out_dir).context(
+                    "Unable to create directory for the resulting template of the schema.",
+                )?;
+            }
+            // saving the template of the schema to the file
+            std::fs::write(out, format!("{}", schema))
+                .context("Unable to write the template of the schema output.")?;
+        }
```

Comment:
```suggestion
            // saving the template of the schema to the file
            let mut out_file = std::fs::File::create(out).context("Unable to create the output file.")?;
            write!(&mut out_file, "{}", schema).context("Unable to write template of the schema output.")?;
        }
```

To avoid creating a string.